spec/features/web_intake/new_single_filer_spec.rb

```diff
@@ -27,6 +25,9 @@
     check "2019"
     click_on "Continue"
 
+    intake = Intake.last
+    expect(intake.client.tax_returns.pluck(:year).sort).to eq [2017, 2019]
```

Comment:
IMHO this is best on a controller spec, not this feature spec. Not a blocker.

Reply:
I was thinking of moving the feature specs in a direction that includes some extra checks, and this was a step in that direction. Do you think thats a good direction? If so, do you think this check should still be moved out of the feature check?